README.md

```diff
@@ -20,6 +20,8 @@
 Marko v3 requires [Marko Widgets v6](https://github.com/marko-js/marko-widgets)\
 Marko v4 requires [Marko Widgets v7](https://github.com/marko-js/marko-widgets/tree/v7)*
 
+*Note: Because eBayUI Core componeents utilize Marko flags you must add `<lasso-page/>` to any page which will have core components. eBayUI Core presently relies on the `skin-ds6` flag to toggle designs to use the Design System 6 version of Skin.*
```

Comment:
Typo on 1st line... `componeents` should be `components`

Reply:
Not necessary, but 1st sentence is easier for me to read if we add a comma after `utilize Marko flags,`

Reply:
Well, it's a singular statement that doesn't need a break. (I had the unfortunate life of having a trained English teacher as a father.)

How about this? (It's essentially now two thoughts.)

>eBayUI Core components utilize Marko flags, and therefore require `<lasso-page/>` to be added to any page which will have core components.

Reply:
That works :)